mayastor/src/replica.rs

```diff
@@ -508,9 +438,9 @@ impl Iterator for ReplicaIter {
 }
 
 pub(crate) async fn create_replica(
-    args: CreateReplicaRequest,
-) -> Result<CreateReplicaReply, RpcError> {
-    let want_share = match ShareProtocolReplica::from_i32(args.share) {
+    args: grpc::CreateReplicaRequest,
```

Comment:
can you just add use grpc:: { .... } at the top?

Reply:
I would have to explicitly import around 12 definitions that I find tedious and needlessly chatty. Another reason is that Replica definition from grpc conflicts with Replica object definition. That one I would have to rename to grpcReplica while importing it. All in all using grpc namespace looks like a better way in this case.